balloc/balloc.c

```diff
@@ -156,6 +156,13 @@ static bool is_spare(uint64_t alloc_flags);
 static bool is_normal(uint64_t alloc_flags);
 static bool is_any(uint64_t alloc_flag);
 
+static int index_hash(struct m0_balloc_super_block *sb,
+			m0_bcount_t block_offset)
+{
+	int grp_no;
+        grp_no = (block_offset - 1) / sb->bsb_groupsize;
```

Comment:
Indentation issue.

More, maybe we don't need to declare variable "grp_no" at all.

Reply:
Done